lang/cy.json

```diff
@@ -1346,11 +1346,12 @@
     "youHaveEnteredNameWhichMatchesSomeone": "Rydych chi wedi rhoi enw sy’n cyfateb i rywun a enwir ar atwrneiaeth arhosol (LPA) {{.DonorFullName}}.",
     "yourConfirmedIdentityDetailsMatchSomeone": "Mae eich manylion adnabod sydd wedi’u cadarnhau yn cyfateb i rywun a enwir ar atwrneiaeth arhosol (LPA) {{.DonorFullName}}.",
     "personConfirmYouWitnessedTheDonorSign": "{{.FullName}}, cadarnhewch eich bod chi wedi bod yn dyst i’r rhoddwr yn llofnodi ei LPA",
-    "theseAreTheDetailsDonorProvidedCheck": "Dyma’r manylion y gwnaeth {{.DonorFullNames}} eu darparu ar eich cyfer chi.  Rhaid i chi wirio eu bod yn gywir cyn i chi fwrw ymlaen.",
+    "theseAreTheDetailsDonorProvidedCheck": "Welsh {{.DonorFullName}}",
```

Comment:
If I'm reading the English diff correctly it looks like this can stay as it is as it's just the var name that's changed.